ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -693,6 +697,20 @@ def _get_matching_session_infos(self, instrument_type_id: str) -> List[SessionIn
             info for info in self._session_info if instrument_type_id == info.instrument_type_id
         ]
 
+    def _to_resolved_pin_or_relay_names(
```

Comment:
I guess the other generic methods in this file were named in the template `_to_abc_def` to denote that the method converts the data structure. Here, we're manipulating the passed-in data. Therefore, I think a better name would be `get_resolved_pin_or_relay_names`.

Reply:
Make sense. Updated.